SourceX/miniwin/misc_msg.cpp

```diff
@@ -260,7 +260,9 @@ WPARAM keystate_for_mouse(WPARAM ret)
 
 bool false_avail(const char *name, int value)
 {
+#ifndef __vita__
```

Comment:
Any reason to disable this on Vita?
Perhaps we should add the events we should ignore instead.

Reply:
Added comment.
Yeah, ignoring events would be better, but i thing that's outside of this pullrequest scope.